modules/table/src/test/java/org/apache/ignite/internal/table/distributed/replication/ZonePartitionReplicaListenerTest.java

```diff
@@ -1466,6 +1464,11 @@ void writeIntentSwitchForCompactedCatalogTimestampWorks(boolean commit) {
         HybridTimestamp beginTs = beginTimestamp(txId);
         HybridTimestamp commitTs = clock.now();
 
+        // We have to force push clock forward because we will invoke listener directly bypass ReplicaManager or MessageService, so clock
+        // won't be updated if the test will compute too fast for physical clock ticking and then we may have equal clock#current and the
+        // given above commit timestamp.
+        clock.update(clock.now().addPhysicalTime(10));
```

Comment:
Why 10? Would just `clock().now()` not be sufficient, without any additions?

Reply:
Changed to `clock#update` with commit timestamp as the argument, the method will increment clock value that will be guarantee later than the commit timestamp.